CHANGELOG.md

```diff
@@ -4,9 +4,67 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.3.0
```

Comment:
https://github.com/nginxinc/nginx-gateway-fabric/pull/1879
we need to mention this as a change ?
@sjberman  I see it was marked as skip change log though

Reply:
The addition of these platforms was never released.

Reply:
we added support and then removed before releasing